Add Japanese and interpolate polyglot phrases

ayalan opened the pull request:
Some polyglot phrases required interpolation to make sense in other languages (Japanese).
- Japanese language is not complete yet in this PR.

jjeffryes commented:
@ayalan let us know when the translation is complete and synced with master, and we'll merge this in. I took a look and tested it, your changes look good. Nice use of variables inside the phrases.

ayalan commented:
@jjeffryes thank you. i18n is almost complete but there are a few more cases where I
will change phrases to use interpolation to be grammatically correct.

ayalan committed:
Add Japanese i18n

ayalan committed:
Add missing phrases and interpolation to other languages

ayalan committed:
Update polyglot phrases

ayalan committed:
Add missing =

ayalan committed:
Fix Japanese langCode

ayalan committed:
Use interpolation in direct payment option during buy flow

ayalan committed:
Splitting polyglot definition for Description

The word for description for a person and a thing are different in Japanese, so I created a specific yourDescription in the onboarding section. Other languages were filled with best effort using the previously used version of Description.

ayalan committed:
Japanese i18n adjustments

ayalan committed:
Use polyglot for Y/N in item editor form

ayalan committed:
Remove duplicate polyglot keys

ayalan committed:
Merge branch 'master' of https://github.com/OpenBazaar/OpenBazaar-Client

ayalan commented:
This is ready to be pulled I think! Please check.

ayalan committed:
Fix Japanese i18n based on context.

ayalan committed:
i18nize "PGP Key"
